src/test/java/com/rultor/agents/daemons/EndsDaemonITCase.java

```diff
@@ -103,6 +106,38 @@ public void readsExitCodeCorrectly() throws IOException {
     }
 
     /**
+     * EndsDaemon can deprecate default image.
+     * @throws IOException In case of error
+     */
+    @Test
+    @Ignore
+    public void deprecatesDefaultImage() throws IOException {
+        final Talk talk = new Talk.InFile();
+        FileUtils.write(
+            new File(this.start(talk, "").getAbsolutePath(), "testing"), "12"
+        );
+        new EndsDaemon().execute(talk);
+        for (final String path
+            : talk.read().xpath("/p/entry[@key='merge']/entry[@key='script']")
+        ) {
+            if ("yegor256/rultor".equals(path)) {
```

Comment:
@sebing We need this to not fire when the repo is the actual Rultor repo: https://github.com/yegor256/rultor :)